include/arc_utilities/arc_helpers.hpp

```diff
@@ -586,6 +588,63 @@ namespace arc_helpers
         }
     }
 
+    template<typename ColorType>
+    inline ColorType Multiply(const ColorType& color, const float factor)
+    {
+        ColorType ret;
+        ret.r = TrimColorValue(factor * color.r);
+        ret.g = TrimColorValue(factor * color.g);
+        ret.b = TrimColorValue(factor * color.b);
+        ret.a = TrimColorValue(factor * color.a);
+        return ret;
+    }
+
+    template<typename ColorType>
+    inline ColorType Multiply(const float factor, const ColorType& color)
+    {
+        return color * factor;
+    }
+
+    template<typename ColorType>
+    inline ColorType Add(const ColorType& c1, const ColorType& c2)
+    {
+        ColorType ret;
+        ret.r = TrimColorValue(c1.r + c2.r);
+        ret.g = TrimColorValue(c1.g + c2.g);
+        ret.b = TrimColorValue(c1.b + c2.b);
+        ret.a = TrimColorValue(c1.a + c2.a);
```

Comment:
Adding the alpha channels is unexpected. Do not add the alpha channels.
One would assume the inverse of `Add` would be a similar `Subtract`, but that would make any two opaque colors return with `ret.a=0`.

Reply:
Agreed that straight addition makes no sense. What do we do if the two inputs have different alpha values?

Reply:
Upon discussion, removed `Add` as we could not determine a clear implementation whose behaviour is obvious.